src/main/materialized_query_result.cpp

```diff
@@ -39,6 +40,17 @@ string MaterializedQueryResult::ToString() {
 	return result;
 }
 
+string MaterializedQueryResult::ToBox(ClientContext &context, const BoxRendererConfig &config) {
+	if (!success) {
+		return GetError() + "\n";
+	}
+	if (!collection) {
```

Comment:
Do we not want to throw an exception here instead?

Reply:
In general results should not throw exceptions because they might be used in places where those are unexpected.